Blockzilla/Modules/WebView/WebViewController.swift

```diff
@@ -62,6 +62,13 @@ class WebViewController: UIViewController, WebController {
 
     var printFormatter: UIPrintFormatter { return browserView.viewPrintFormatter() }
     var scrollView: UIScrollView { return browserView.scrollView }
+    private enum ScriptHandlers: String {
```

Comment:
Usually whenever I add a nested type, I will do it directly below the parent type declaration to separate it from the body of the class.

```swift
class WebViewController: UIViewController, WebController {
    private enum ScriptHandlers: String {
   ...
```

Another method that I've seen is using an extension at the bottom of the file:

```swift
extension WebViewController {
    fileprivate enum ScriptHandlers: String { }
}
```